crates/tauri-utils/src/config.rs

```diff
@@ -2468,7 +2468,8 @@ pub struct SecurityConfig {
   pub pattern: PatternKind,
   /// List of capabilities that are enabled on the application.
   ///
-  /// If the list is empty, all capabilities are included.
+  /// By default (leftout or empty list), all capability files from `./capabilities/` (relative to your `build.rs`) are included,
```

Comment:
```suggestion
  /// By default (unset or empty list), all capability files from `./capabilities/` (relative to your `build.rs`) are included,
```

most nit-picky nitpick so far i think 😂 i think left-out is too "complicated" of a word here (#support hurt me), something like unset or not set or something may be better imo.

Also, is it really relative to build.rs ? What if the user moves build.rs into src (setting it in Cargo.toml as well) do we generate/read those files to/from src/capabilities/ then? genuine question, never thought about this before.

Reply:
> Also, is it really relative to build.rs ?

I have not even tried that before 😂, our code in `tauri-build` just uses `./capabilities/**` to glob the files, I need to check if that's relative to `build.rs` or `Cargo.toml`

Reply:
Went with `not set` 😂

Reply:
It's relative to `Cargo.toml`, updated, good call 😂

> https://doc.rust-lang.org/cargo/reference/build-scripts.html#:~:text=In%20addition%20to%20environment%20variables%2C%20the%20build%20script%E2%80%99s%20current%20directory%20is%20the%20source%20directory%20of%20the%20build%20script%E2%80%99s%20package.